tracer/build/_build/PrepareRelease/SetAllVersions.cs

```diff
@@ -166,6 +166,10 @@ public void Run()
                 "src/WindowsInstaller/WindowsInstaller.wixproj",
                 WixProjReplace);
 
+            SynchronizeVersion(
```

Comment:
In `shared/src/msi-installer/WindowsInstaller.wixproj`, I think you'll need to also modify the following line on line 20:

```xml
    <OutputName>datadog-dotnet-apm-1.28.6-$(Platform)-profiler-beta</OutputName>
```

and break it up into the two following lines so that our regex will replace the first one:

```xml
    <OutputName>datadog-dotnet-apm-2.0.1-$(Platform)</OutputName> <!-- -The regex recognizes this line ->
    <OutputName>$(OutputName)-profiler-beta</OutputName>
```

Reply:
Oh right.
done in 798b793